blocks/curated-cards/curated-cards.css

```diff
@@ -1,194 +1,173 @@
-        gap: 28px;
-
-        img {
-          object-fit: cover;
-          border-radius: 5px;
-        }
-
-        .curated-cards-card-text-wrapper {
-          p {
-            margin: 0;
-            font-weight: 700;
-            font-size: var(--body-medium);
-            line-height: 140%;
-            color: var(--color-white);
-          }
-
-          .curated-cards-card-event {
-            font-weight: 700;
-            font-size: var(--body-tiny);
-            letter-spacing: 0.06em;
-            text-transform: uppercase;
-            color: var(--neutrals-50);
-            margin-top: 15px;
-          }
-
-        }
-      }
-    }
-  }
+.curated-cards .feature-card .feature-card-content {
+  position: absolute;
+  bottom: 20px;
+  left: 20px;
+  width: calc(100% - 40px);
+  color: var(--color-white);
+  padding: 20px;
+  box-sizing: border-box;
+}
+
+.curated-cards .feature-card .feature-card-content .feature-card-content-heading {
+  font-size: var(--h1-base);
+  margin: 0 0 10px;
+  line-height: 58px;
+  font-weight: 700;
+}
+
+.curated-cards .feature-card .feature-card-content button {
+  position: absolute;
+  right: 20px;
+  bottom: 20px;
+  padding: 10px 20px;
+  font-size: var(--body-medium);
+  width: 182px;
+  height: 55px;
+  background: var(--color-white);
+  border-radius: 100px;
+  color: var(--primary-blue-80);
+}
+
+.curated-cards.list {
+  background-color: var(--primary-blue-100);
+  border-radius: 0 0 10px 10px;
+  padding: 25px;
+  padding-block-start: 0;
+  margin-top: -3px;
+}
+
+.curated-cards.list .curated-cards-line-div {
+  display: flex;
+  align-items: center;
+  padding-block-end: 40px;
+  gap: 16px;
+}
+
+.curated-cards.list .curated-cards-line-div .line-text {
+  color: var(--text-on-dark-emphasys, #FFFFFFE5);
+  padding-left: 10px;
+  font-size: 12px;
+  font-weight: 700;
+}
+
+.curated-cards.list .curated-cards-line-div .line {
+  height: 2px;
+  width: calc(100% - 140px);
+  background: #FFFFFF3D;
+  margin: 0 auto;
+}
+
+.curated-cards.list ul {
+  display: grid;
+  padding: 0 10px;
+  margin-top: 0;
+  margin-bottom: 20px;
+  gap: 32px;
+  grid-template-columns: repeat(3, 1fr);
+}
+
+.curated-cards.list ul li {
+  display: grid;
+  grid-template-columns: minmax(30%, 160px) 1fr;
+  padding: 0;
+  gap: 28px;
+}
+
+.curated-cards.list ul li img {
+  object-fit: cover;
+  border-radius: 5px;
+  width: 100%;
+  height: auto;
+}
+
+.curated-cards.list ul li .curated-cards-card-text-wrapper p {
+  margin: 0;
+  font-weight: 700;
+  font-size: var(--body-medium);
+  line-height: 140%;
+  color: var(--color-white); 
+}
+
+.curated-cards.list ul li .curated-cards-card-text-wrapper a{
+  text-decoration: none;
+}
+
+.curated-cards.list ul li .curated-cards-card-text-wrapper .curated-cards-card-event {
+  font-weight: 700;
+  font-size: var(--body-tiny);
+  letter-spacing: 0.06em;
+  text-transform: uppercase;
+  color: var(--neutrals-50);
+  margin-top: 15px;
 }
 
 @media (width <=1024px) {
```

Comment:
Tab is <1024

Reply:
Using < 1024px would exclude devices with exactly 1024px width, so it's generally safer to use <= 1024px (expressed as max-width: 1024px in CSS) to cover all cases. @mkbansal1